content/chainguard/chainguard-images/working-with-images/security-advisories/managing-advisories/index.md

```diff
@@ -0,0 +1,165 @@
+                                                                 2024-05-15T04:39:53Z fixed (2.39-r5)
+      CGA-7fr2-v9gg-pg28 (CVE-2024-33600, GHSA-jv3g-6pg3-v9j8)   2024-05-14T11:00:51Z detected (glibc)
+                                                                 2024-05-15T04:40:16Z fixed (2.39-r5)
+      CGA-vjpp-g2r4-c8gc (CVE-2024-33601, GHSA-f4cf-2w52-c853)   2024-05-14T11:00:53Z detected (glibc)
+                                                                 2024-05-15T04:40:32Z fixed (2.39-r5)
+      CGA-wfxf-8642-f6f5 (CVE-2024-33602, GHSA-f4pv-q5f7-2h55)   2024-05-14T11:00:56Z detected (glibc)
+                                                                 2024-05-15T04:40:42Z fixed (2.39-r5)
+
+…
+```
+
+From this snapshot, we can get an idea of the timeline of a vulnerability’s remediation process. The first CVE displayed, CVE-2024-33599, was detected on 5/14/2023 and remediated within the next 24 hours. Similar results are shown for other advisories, including the versions of the package in which the vulnerability was remediated.
+
+We encourage you to experiment with these flags to see what information you can gather from your various searches.
+
+## Creating and Updating Advisories
+
+To begin creating and updating security advisories, we will need to navigate back to the package repository we cloned.
+
+```shell
+cd ../os 
+```
+
+Here, we will run the `wolfictl advisory create` command to begin the process of adding a new advisory. This command will respond with a few prompts in the terminal requesting input of information used to create the advisory.
+
+First, we will be asked to enter the package in which the CVE was found. For our demonstration we will use the `glibc` package. Then, we will be asked for the vulnerability ID that we wish to make an advisory for. This ID must be a CVE, CGA, GHSA, or Go vulnerability ID. Finally, we will be asked what the status for the advisory should be, from one of the following options:
+- detection (Under investigation)
+- true-positive-determination (Affected)
+- fixed (Fixed)
+- false-positive-determination (Not affected)
+- fix-not-planned (Fix not planned)
+- pending-upstream-fix (Pending upstream fix)
+
+To learn more about the meanings of each of these identifiers, please refer to our [documentation on event types](https://github.com/wolfi-dev/advisories/blob/main/docs/event_types.md). The following shows an example of the `wolfictl advisory create` command in action. **Please note that the vulnerability exemplified here is an arbitary CVE ID for demonstration purposes and does not reflect an actual vulnerability found in the `glibc` package.**
+
+```sh
+Auto-detected distro: Wolfi
+
+Package: glibc 
+Vulnerability: CVE-2024-57230 
+Type: 
+  detection
+```
+
+We can now check that our advisory has been successfully added with the `wolfictl advisory diff` command. This will show us what advisories have been modified by us.
```

Comment:
show, don't just tell! Rather than saying the reader can run the command, just tell them to run it and show the output.